TicketsDemo.Xml/Interfaces/ITrainXMLWriter.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TicketsDemo.Xml.Interfaces
+{
+    public interface ITrainXMLWriter
```

Comment:
rename to ITrainsImporter.

implementation to be renamed EFIntoXMLTrainsImporter